Makefile

```diff
@@ -22,6 +23,8 @@ install: $(SCHEMA_FILES)
 	createdb userdb
 	$(foreach file,$(SCHEMA_FILES),psql -v ON_ERROR_STOP=1 -h localhost -d userdb -f $(file);)
 
+reserve: usernames.txt usernames.sql
+	psql -v ON_ERROR_STOP=1 -h localhost -d userdb -f usernames.sql
```

Comment:
Could you add usernames.sql to SCHEMA_FILES so that it's done by `install` like everything else?